tests/ui_tests/app/page/cluster/page.py

```diff
@@ -211,6 +220,97 @@ class ClusterConfigPage(ClusterPageMixin):
     """Cluster page config menu"""
 
     MENU_SUFFIX = 'config'
+    MAIN_ELEMENTS = [
+        ObjectPageLocators.title,
+        ObjectPageLocators.subtitle,
+        ClusterMainLocators.text,
+        ClusterConfigLocators.description_input,
+        ClusterConfigLocators.search_input,
+        ClusterConfigLocators.advanced_chbx,
+        ClusterConfigLocators.save_btn,
+        ClusterConfigLocators.history_btn,
+    ]
+
+    def fill_search_input(self, text: str):
+        self.find_element(ClusterConfigLocators.search_input).clear()
+        self.send_text_to_element(ClusterConfigLocators.search_input, text)
+
+    def click_save_btn(self):
+        self.find_and_click(ClusterConfigLocators.save_btn)
+
+    def click_history_btn(self):
+        self.find_and_click(ClusterConfigLocators.history_btn)
+
+    def select_comparing_history_by_index(self, index_from: int, index_to: int):
+        self.click_history_btn()
+        self.wait_element_visible(ClusterConfigLocators.HistoryRow.history_select).click()
+
+        def click_option(index):
+            self.wait_element_visible(ClusterConfigLocators.HistoryRow.option)
+            self.find_elements(ClusterConfigLocators.HistoryRow.option)[index].click()
```

Comment:
Maybe make this method private at the class level. Can't see why it should be defined inside the method.

Reply:
removed